src/cmd/tools/dtest/docker/harness/resources/coordinator.go

```diff
@@ -330,6 +320,26 @@ func (c *coordinator) WriteCarbon(
 	// return nil
 }
 
+func makePostRequest(logger *zap.Logger, url string, body proto.Message) (*http.Response, error) {
+	data := bytes.NewBuffer(nil)
+	if err := (&jsonpb.Marshaler{}).Marshal(data, body); err != nil {
+		logger.Error("failed to marshal", zap.Error(err))
+
+		return &http.Response{}, fmt.Errorf("failed to marshal: %w", err)
```

Comment:
Maybe just return `nil` in these error cases?

Reply:
https://github.com/m3db/m3/pull/2849/commits/f537edccd8d8b4efa432e197d58f7a8d3f98a3c0